nbs/sidebar.yml

```diff
@@ -2,3 +2,4 @@ website:
   sidebar:
```

Comment:
_Commented on notebook `nbs/sidebar.yml` Cell 1 Line 2_
```python
website:
  sidebar:
```
comment on sidebar

_Respond and view the context [here](https://gitnotebooks.com/GitNotebooks/docs/pull/2)._